src/sanitizer.js

```diff
@@ -26,16 +26,45 @@ export const sanitizeDocFragment = function _sanitizeDocFragment(
  * @private
  */
 export const _normalizeConfig = function _normalizeConfig(config) {
-  if (!config) {
+  if (
+    config &&
+    Object.keys(config).length === 0 &&
+    config.constructor === Object
+  ) {
     config = {};
   }
-  const allowElements = config.allowElements || DEFAULT_ALLOWED_ELEMENTS;
-  const allowAttributes = config.allowAttributes || DEFAULT_ALLOWED_ATTRIBUTES;
-  const blockElements = config.blockElements || DEFAULT_BLOCKED_ELEMENTS;
-  const dropElements = config.dropElements || DEFAULT_DROP_ELEMENTS;
-  const dropAttributes = config.dropAttributes || DEFAULT_DROP_ATTRIBUTES;
-  const allowComments = !!config.allowComments;
-  const allowCustomElements = !!config.allowCustomElements;
+
+  let normalizedConfig = {};
+  // TODO https://github.com/mozilla/sanitizer-polyfill/issues/29
+  for (let [configurationElementList, elements] of Object.entries(config)) {
+    if (SUPPORTED_CONFIGURATION_LISTS.has(configurationElementList)) {
```

Comment:
We currently don't normalize configuration keys by camelcasing them. For example, if the user passes the following config object `{'allowelements': ['a']}` to `_normalizeConfig`, the output would be the default configuration object.

I think it's on the user to use the correct configuration keys rather than trying to camelcase all keys in the config object we receive from the user. What is your opinion here? :)

Reply:
yep. (We _could_ log to the console whenver we use a list from the defaults but that might be a bit noisy)